tox.ini

```diff
@@ -20,6 +20,14 @@ commands =
    poetry install -v {env:INSTALL_OPTS}
    poetry run pytest --quiet --cov=generated/nidaqmx --cov-append --cov-report= --junitxml=test_results/system-{envname}.xml {env:PYTEST_OPTS} {posargs}
 
```

Comment:
The new envs have no value on Linux, so let's restrict them to Windows. Add this to the `[testenv]` section:
```
platform =
   nicaiu: win32
   nicai_utf8: win32
```